CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 *********
 
+1.0.4 -- 2024-03-09
+===================
+added optional support for the `pybase64` library to improve performance
+
```

Comment:
Blocking: Revert.
I think there will be quite a few release notes added when ever we get around to pushing the next release of this out.
It will also probably be a new major release, since we will be dropping Python 2.
While we maintain this library, it has been ~6 years since our last release,
as it is such a small library that has not needed any changes.

```suggestion
```

Reply:
ok  i removed  it